src/diffusers/models/autoencoders/autoencoder_kl.py

```diff
@@ -337,7 +341,7 @@ def blend_h(self, a: torch.Tensor, b: torch.Tensor, blend_extent: int) -> torch.
             b[:, :, :, x] = a[:, :, :, -blend_extent + x] * (1 - x / blend_extent) + b[:, :, :, x] * (x / blend_extent)
         return b
 
-    def tiled_encode(self, x: torch.Tensor, return_dict: bool = True) -> AutoencoderKLOutput:
+    def tiled_encode(self, x: torch.Tensor) -> torch.Tensor:
```

Comment:
maybe, if we concern breaking, we can deprecate tiled_encode and make a new one called `_tiled_encode`

Reply:
yeah, actually prefer to do that, I do see some usage of `titled_encode()`; also our current implementation of `titled_encode` is something can be used on its own, the new one is more like a private method that has to be called inside `_encode`

Reply:
Sounds good, I'll make a new method